app/src/main/java/net/squanchy/proximity/near/NearITProximityProvider.java

```diff
@@ -0,0 +1,34 @@
+package net.squanchy.proximity.near;
+
+import android.content.Context;
+import android.util.Log;
+
+import net.squanchy.R;
+import net.squanchy.proximity.ProximityProvider;
+
+import it.near.sdk.Geopolis.Beacons.Ranging.ProximityListener;
+import it.near.sdk.NearItManager;
+
+public class NearITProximityProvider implements ProximityProvider{
+
+    NearItManager nearItManager;
+
+    public NearITProximityProvider(Context context) {
+        Log.d("NearITProximityProvider", "new instance created");
```

Comment:
Please, remove the logging from the constructor. Bear in mind we use [timber](https://github.com/JakeWharton/timber) for logging, in case you should need to log.